tests/components/group/test_config_flow.py

```diff
@@ -173,19 +176,22 @@ async def test_options(
         result["flow_id"],
         user_input={
             "entities": members2,
+            "hide_members": False,
```

Comment:
`hide_members` is `vol.Required` with a default value.
How come the options flow fails if `hide_members` is not passed, but the config flow doesn't?